resources/planwise/sql/datasets2.sql

```diff
@@ -43,7 +43,18 @@ SELECT s2.id, s2.name, s2.lat, s2.lon, s2.capacity, s2.type, s2.tags, s2c.raster
           AND version = :version
           AND s2."the_geom" @ (SELECT "the_geom" FROM regions WHERE id = :region-id)
           AND s2c.algorithm = :algorithm
-          AND s2c.options = :options;
+          AND s2c.options = :options
+         /*~ (if (seq (:tags params)) */
+          AND s2.tags::tsvector @@ :tags::tsquery;
+         /*~ ) ~*/;
+
+-- :name db-count-sites-with-tag :? :1
+SELECT COUNT(*) FROM sites2
+    WHERE "dataset-id" = :dataset-id
+    AND version = :version
```

Comment:
For consistent results, please add the region clipping filter, ie.
```
AND s2."the_geom" @ (SELECT "the_geom" FROM regions WHERE id = :region-id)
```
Of course, this needs to be conditional to the `:region-id` parameter not being `nil`.